### ISSUE TITLE
(GPU) cannot run vl_test_nnlayers when GPU is enabled

### ISSUE THREAD

**Issue by johnny5550822** [vlfeat/matconvnet#41]
I compiled the code with GPU option enable and there are no error messages, however, when I try to run vl_test_nnlayers, it gave me this:

>> vl_test_nnlayers(true)
test number 1
testing vl_nnsoftamxloss multiple images convolutional
test number 2
testing vl_nnloss multiple images convolutional
testing vl_nnloss multiple images
testing vl_nnloss
test number 3
testing vl_nnsoftmax
test number 4
testing vl_nnconv with fully connected identity filters
Invalid MEX-file
'~/matconvnet-master/matlab/mex/vl_nnconv.mexa64':
libcudart.so.6.5: cannot open shared object file: No such
file or directory

I am using CUDA 6.5 and gcc 4.4. 

Thanks!

**Comment by lenck** [comment]
Hi, this is sually caused by missing paths in `LD_LIBRARY_PATH` environment variable - the linker basically cannot find the `licudart` library. This usually does not happen when CUDA is installed through package manager, I suppose....
How to set it up is described e.g. [here](http://docs.nvidia.com/cuda/cuda-getting-started-guide-for-linux/#post-installation-actions) and you can put those commands in your `~/.bashrc` to make these settings permanent for each bash session.
Basically it is about running these commands before starting Matlab in the same terminal session:
```
$ export PATH=/usr/local/cuda-6.5/bin:$PATH
$ export LD_LIBRARY_PATH=/usr/local/cuda-6.5/lib64:$LD_LIBRARY_PATH
```
(just copy-paste from the NVIDIA website ;)).

**Comment by johnny5550822** [comment]
Great, thanks! I tried to solve it before the reply and it worked now.

**Comment by Tgaaly** [comment]
thanks @lenck - this helped me!!
added those lines for cuda-7.0 in my bashrc and ran matlab from command line.

**Comment by cheer37** [comment]
How can i fix this problem on windows?
I installed cuda7.0 by self installing package.
Thanks in advance.
